public/css/me.css

```diff
@@ -52,49 +65,98 @@ button:hover {
 
 .blurb {
 	margin-bottom: 20px;
-}
+} */
 
 /* APPLICATION STATUS */
 
-.background {
+/* .background {
 	position: relative;
 	width: 510px;
 	margin: 40px auto;
 	padding: 20px 70px;
 	border-radius: 10px;
-}
+} */
 
-.background h1 {
+/* .background h1 {
 	font-family: "Source Code Pro", monospace;
 	font-weight: 500;
 	font-size: 36px;
 	color: #2b2b2b;
 	letter-spacing: -1.5px;
-}
+} */
 
 .status-container {
 	display: inline-block;
 	margin: 20px 0;
-	padding: 10px;
-	background-color: #fafafa;
-	border-radius: 10px;
+	padding-top: 40px;
+	padding-bottom: 50px;
+	padding-left: 70px;
+	padding-right: 70px;
+	background-color: var(--white);
+	border-radius: 40px;
 }
 
 .status {
-	padding: 50px 30px;
-	border-radius: 10px;
-	width: 320px;
+	padding: 60px 20px;
+	border-radius: 40px;
+	width: 400px;
+	height: 200px;
+	/* display: flex;
+	flex-wrap: nowrap;
+	align-items: center;
+	justify-content: center; */
+}
```

Comment:
To keep the status box centered when the status description gets long (and if you don't want to use `flex`), you can use `margin: 20px auto 0;`. For a block-displayed tag with a fixed width, `auto` for side margins will center it